docs/operating-eck/troubleshooting/common-problems.asciidoc

```diff
@@ -214,3 +214,10 @@ elastic-cloud-eck.v1.4.0       Elasticsearch (ECK) Operator   1.4.0       elasti
 If you are using one of the affected versions of OLM and upgrading OLM to a newer version is not possible then ECK
 can still be upgraded by uninstalling and reinstalling it. This can be done by removing the `Subscription` and both `ClusterServiceVersion` resources and adding them again.
 On OpenShift the same workaround can be performed in the UI by clicking on "Uninstall Operator" and then reinstalling it through OperatorHub.
+
+[id="{p}-{page_id}-version-downgrade"]
+== If you upgraded Elasticsearch to the wrong version
+If you accidentally upgrade one of your Elasticsearch clusters to a version that does not exist or a version to which a direct upgrade is not possible from your currently deployed version, a validation will prevent you from going back to the previous version.
+The reason for this validation is that ECK will not allow downgrades as this is not supported by Elasticsearch and once the data directory of Elasticsearch has been upgraded there is no way back to the old version.
+
+The two scenarios described above however are exceptions because Elasticsearch never started up successfully. If you annotate the Elasticsearch resource with `k8s.eck.elastic.co/disable-downgrade-validation` ECK will allow you to go back to the old version at your own risk. Please remove the annotation afterwards to prevent accidental downgrades and reduced availability.
```

Comment:
We mention annotating the ES resource with `k8s.eck.elastic.co/disable-downgrade-validation`, but the validation below also checks the `value == "true"`, and not just the existence of the key.

Reply:
Yes I forgot to update the documentation when I changed the behaviour 😄